e2e_playwright/st_heading_test.py

```diff
@@ -269,3 +272,16 @@ def test_help_tooltip_works(app: Page):
 
     title_with_help = _get_title_elements(app).nth(1)
     expect_help_tooltip(app, title_with_help, tooltip_text)
+
+
+def test_not_scrolled_on_empty_anchor_tag(app: Page):
+    """Test that the page is not scrolled when the page contains an empty
+    header/anchor tag and no window hash."""
+
+    # Check if the page is still scrolled to the top
+    # after one second timeout.
+    app.wait_for_timeout(1000)
+    scroll_position = app.evaluate("window.scrollY")
+    # Usage of assert is fine here since we just need to verify that
```

Comment:
still confused why assert over expect here?

Reply:
I believe there isn't any [`expect` in playwright](https://playwright.dev/python/docs/test-assertions#list-of-assertions) we could use to validate that the scroll position is still at 0. The [way to evaluate any JavaScript in playwright](https://playwright.dev/python/docs/evaluating) directly returns the result. This allows us to use `assert` to check if the value is as expected. 

Using `assert` in playwright tests can very quickly lead to flakiness. But in this specific case, we don't really need to wait for the `scroll` status to become `0` since this is the default position. We need to ensure it didn't change after the app is loaded.

Reply:
ah i see, okay that makes sense to me!